src/deploy/functions/prepare.ts

```diff
@@ -486,7 +488,12 @@ export async function loadCodebases(
       "functions",
       `Loading and analyzing source code for codebase ${codebase} to determine what to deploy`,
     );
-    const discoveredBuild = await runtimeDelegate.discoverBuild(runtimeConfig, {
+
+    const codebaseRuntimeConfig = shouldUseRuntimeConfig(codebaseConfig)
```

Comment:
worth logging here? maybe if `true`, log a warning that runtime config is going away? tempting to log if `false`, but that would probably get too noisy and might confuse.